docs/conf.py

```diff
@@ -40,6 +40,15 @@
     'jira',
 ]
 
+# When we drop el7, just put classad-attribute unconditionally
+# into extensions. It requires python3, so it doesn't 
+# work on python2 sphinxes, as found on el7.
+
+if (sys.version[0:1] == '3'):
+	extensions.append('classad-attribute')
+        extensions.append('sphinx.ext.autodoc')
+        extensions.append('sphinx_autodoc_typehints')
+
```

Comment:
Could add an `elif sys.version_info.major == 2:` block here that does something like https://stackoverflow.com/a/14463362/602297